Runtime/Components/ConeRayAnglesEstimator/AveragedConeRaySegmentComputation.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ubco.ovilab.HPUI.Interaction;
+
+namespace ubco.ovilab.HPUI
+{
+    /// <summary>
+    /// Computes the cone ray angles by averaging accorss frames.
+    /// </summary>
+    [Serializable]
+    public class AveragedConeRaySegmentComputation : IConeRaySegmentComputation
+    {
+        List<HPUIInteractorRayAngle> IConeRaySegmentComputation.EstimateConeAnglesForSegment(HPUIInteractorConeRayAngleSegment segment, IEnumerable<ConeRayComputationDataRecord> interactionRecords)
+        {
+            Dictionary<(float, float), float> averageRayDistance = new();
```

Comment:
Can we just add a comment above this being like the key is x and z angles, and value is the averaged distance so that there's less confusion in the future? or maybe rename the variable to something like XZAnglesToAvgDistanceDict (maybe come up with a better name :p)

Reply:
https://github.com/ovi-lab/HPUI-Core/pull/32/commits/fd50d38591fec1b6675ca24c40172bbaa0d9c826 resolves this?